code/datums/components/storage/storage.dm

```diff
@@ -772,6 +772,8 @@
 /datum/component/storage/proc/on_alt_click(datum/source, mob/user)
 	if(!isliving(user) || !user.CanReach(parent))
 		return
+	if(istype(parent, /obj/item/storage/lockbox))
+		return
```

Comment:
This prevents alt clicking to open lockboxes, which is a very useful thing when moving around mindshields